src/web/app/pages-student/student-course-details-page/student-course-details-page.component.ts

```diff
@@ -214,4 +218,26 @@ export class StudentCourseDetailsPageComponent implements OnInit {
       return this.tableComparatorService.compare(sortOption, SortOrder.ASC, strA, strB);
     });
   }
+
+  /**
+ * Search Profiles if the value of the search bar is included
+ * in the name or the email of the profile
+ */
+  searchTeammates(): void {
+
+    let inputValue = (<HTMLInputElement>document.getElementById("inputValue")).value;
```

Comment:
Can we try having the input value passed directly into the method instead of selecting my HTML element? See https://stackoverflow.com/questions/54349299/pass-html-input-value-to-angular-process